app/models/peep.rb

```diff
@@ -0,0 +1,10 @@
+class Peep
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :content, Text
+  property :created_at, DateTime
+
+  validates_presence_of :content
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.